lane.py

```diff
@@ -1223,6 +1250,95 @@ def audience_filter_clauses(self, _db, qu):
             clauses.append(LicensePool.data_source_id != gutenberg.id)
         return clauses
 
+    def customlist_filter_clauses(
```

Comment:
I moved this method from `Lane` to `WorkList` without, I think, changing any code.